AU2/frontends/inquirer_cli.py

```diff
@@ -166,6 +166,9 @@ def render(html_component, dependency_context={}):
     # dependent component
     elif isinstance(html_component, AssassinPseudonymPair):
         assassins = [a[0] for a in html_component.assassins]
+        # enforce including defaults, in case they are hidden, so that the user doesn't accidentally remove an assassin

```

Comment:
```suggestion
        # context: Searchable meta-component
        # enforce including defaults, in case they are hidden, so that the user doesn't accidentally remove an assassin
```

?

Reply:
The context here is actually assassins with `hidden` set to `True`, so I guess I should clarify that (though it also works for `Searchable` as well)

Reply:
Actually, I've realised the component doesn't handle the added defaults properly oops. I thought I'd tested this but I must have <kbd>Ctrl</kbd>+<kbd>C</kbd>'d out every time... To fix this the component would need to query `ASSASSINS_DATABASE` to fetch the missing pseuodnym which doesn't seem like something we want it to do. So I'm going to move the inclusion of defaults into `on_event_request_update` instead